src/database.ts

```diff
@@ -3515,6 +3621,7 @@ class Database extends common.GrpcServiceObject {
 promisifyAll(Database, {
   exclude: [
     'batchTransaction',
+    'batchWrite',
```

Comment:
Why do we need to add this method here, this method does not return a promise ?

Reply:
No, this return a readable stream. Hence, adding it here

Reply:
Got it. Its an exclude list.